src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,60 +1,103 @@
+  [
+    <p key="vic"><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>,
+    ['id-card']
+  ]
+];
+
 class AuthApplicationSection extends React.Component {
   render() {
-    let content;
-    const verifiedAccountType = 3;// verified ID.me accounts are type 3
-    const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
-    const idCardAvailable = this.props.userProfile.services.indexOf('id-card') !== -1;
-    if (isVerifiedUser) {
-      content = (
+    const { services } = this.props.userProfile;
+    const availableServices = [];
+    const unavailableServices = [];
+    const isAvailable = services.reduce((acc, service) => {
+      acc[service] = true;
+      return acc;
+    }, {});
+
+    contentConditions.forEach(([content, requiredServices]) => {
+      // Services without any requirements are available to every user.
+      if (!requiredServices) {
+        availableServices.push(content);
+        return;
+      }
+
+      const accessible = requiredServices.length > 1 ?
+        requiredServices.reduce((acc, service) => acc && isAvailable[service], true) :
+        isAvailable[requiredServices[0]];
+      if (accessible) {
+        availableServices.push(content);
+      } else {
+        unavailableServices.push(content);
+      }
+    });
+
+    return (
+      <div className="profile-section medium-12 columns">
+        <h4 className="section-header">Available services</h4>
         <div className="medium-12 columns">
-          <p><span className="label">Your account will allow you to:</span></p>
-          <p><a href="/health-care/apply/">Apply for health care</a></p>
-          <p><a href="/education/apply-for-education-benefits">Apply for education benefits</a></p>
-          <p><a href="/discharge-upgrade-instructions/">Apply for a discharge upgrade if needed</a></p>
-          <p><a href="/health-care/prescriptions">Refill your prescription</a></p>
-          <p><a href="/health-care/messaging">Message your health care team</a></p>
-          <p><a href="/health-care/health-records">Get your VA health records</a></p>
-          <p><a href="/track-claims">Check your claim and appeal status</a></p>
-          <p><a href="/education/gi-bill/post-9-11/ch-33-benefit">Get your Post-9/11 GI Bill statement of benefits</a></p>
-          {idCardAvailable && <p><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>}
+          {
+            !!availableServices.length && (
+              <p><span className="label">Your account will allow you to:</span></p>
+            )
+          }
+          {availableServices}
+          {
+            !!unavailableServices.length && (
+              <p><span className="label">You need to <a href="/verify?next=/profile">verify your identity</a> in order to:</span></p>
+            )
+          }
+          {unavailableServices}
           <p><strong>Note:</strong> If you have a less than honorable discharge, you can apply for an upgrade. If your
```

Comment:
@ncksllvn || @bshyong: Since there have been a few changes to this page from Discharge Wizard, I wanted to check if I resolved the merge conflicts properly, and everything is in the right place.